enhancements/update/update-blocker-lifecycle/README.md

```diff
@@ -75,13 +77,13 @@ Write tooling that automatically:
 
 The following statement (or a link to this section) can be pasted into bugs when adding `ImpactStatementRequested`:
 
-We're asking the following questions to evaluate whether or not this bug warrants changing update recommendations from either the previous X.Y or X.Y.Z.
-The ultimate goal is to avoid delivering an update which introduces new risk or reduces cluster functionality in any way.
+We're asking the following questions to evaluate whether or not OCPBUGS-# warrants changing update recommendations from either the previous X.Y or X.Y.Z.
+The ultimate goal is to avoid recommending an update which introduces new risk or reduces cluster functionality in any way.
 In the absence of a declared update risk (the status quo), there is some risk that the existing fleet updates into the at-risk releases.
 Depending on the bug and estimated risk, leaving the update risk undeclared may be acceptable.
 
-Sample answers are provided to give more context and the ImpactStatementRequested label has been added to this bug.
-When responding, please remove ImpactStatementRequested and set the ImpactStatementProposed label.
+Sample answers are provided to give more context and the `ImpactStatementRequested` label has been added to OCPBUGS-#.
+When responding, please move this ticket to `Code Review`.
```

Comment:
I do not see the separate Jira (spike) workflow anywhere in this PR. Do you want to add it in a separate PR.

Reply:
This should be the only Jira-side action that the assignee needs to take, so I've included it here (replacing their previous Bugzilla-side action).  From the PR topic post:

> I've also left off the details of creating the component-team side ticket, where we currently want:
> ...
> Those seem like they can remain internal, mutable details for now, and not get frozen out in the enhancement yet.

And yeah, that's all stuff on the updates-team/robots side that I didn't want to commit to in this pull request.